packages/cli/src/cli/index.ts

```diff
@@ -73,9 +73,7 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
 
   program
     .command("build")
-    .description(
-      "Compile contracts and detect implemented thirdweb contract extensions",
-    )
+    .description("Compile contract and detect thirdweb contract extensions")
```

Comment:
`contract` -> `contracts` (this can be used on many contracts at once).

Reply:
`build` also still feels like the wrong command here, but `detect` definitely wasn't right either, so this is fine for now. `info`, `compile` are other ideas

Reply:
Why does build feel wrong? info / detect don't make sense. I can see compile as the function but feels like in the context of an app build is more in line with what people think about